playbook.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
+
+  tasks: 
+    - shell: echo hello world
```

Comment:
* [**E301**] Commands should either read information (and thus set changed_when) or not do something if it has already been done (using creates/removes) or only do it if another check has a particular result (when)

* [**E305**] Shell should only be used when piping, redirecting or chaining commands (and Ansible would be preferred for some of those!)

* [**E502**] All tasks should have a distinct name for readability and for --start-at-task to work